### ISSUE TITLE
Compiler error for Cheatcodes demo code

### ISSUE THREAD

**Issue by sbvegan** [foundry-rs/book#223]
I'm new to foundry and was reading through the book when I ran into an issue on the Cheatcodes section: https://book.getfoundry.sh/forge/cheatcodes.html

When I got to the section that says, "Let's make sure that someone who is definitely not the owner can't increment the count" I added the `CheatCodes` interface and the `testFailIncrementAsNotOwner` test I got the following error:

```
soyboy@sbv:~/code/hello_foundry$ forge test
[⠒] Compiling...
Error: 
   0: Compiler run failed
      DeclarationError: Undeclared identifier.
        --> /home/soyboy/code/hello_foundry/test/Cheatcodes.t.sol:41:9:
         |
      41 |         cheats.prank(address(0));
         |         ^^^^^^

      
   0: 

Location:
   cli/src/compile.rs:149

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

This is what my test file looks like: 

```
pragma solidity 0.8.10;

import "ds-test/test.sol";

error Unauthorized();

contract OwnerUpOnly {
    address public immutable owner;
    uint256 public count;

    constructor() {
        owner = msg.sender;
    }

    function increment() external {
        if (msg.sender != owner) {
            revert Unauthorized();
        }
        count++;
    }
}

interface CheatCodes {
    function prank(address) external;
}

contract OwnerUpOnlyTest is DSTest {
    OwnerUpOnly upOnly;

    function setUp() public {
        upOnly = new OwnerUpOnly();
    }

    function testIncrementAsOwner() public {
        assertEq(upOnly.count(), 0);
        upOnly.increment();
        assertEq(upOnly.count(), 1);
    }

    function testFailIncrementAsNotOwner() public {
        cheats.prank(address(0));
        upOnly.increment();
    }
}

```

Where do the cheatcodes come from and how are they properly used in a test?

**Comment by ZeroEkkusu** [comment]
You are right, the `CheatCodes cheats = CheatCodes(HEVM_ADDRESS);` part is missing.
Add it to your `OwnerUpOnlyTest` contract, below `OwnerUpOnly upOnly;`.

This will grab the address from `HEVM_ADDRESS` in `DSTest` you inherited and create `cheats` instance via which you can access cheatcodes.

This will be fixed in #207.

**Comment by sbvegan** [comment]
Ah thank you for the missing piece!
